specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-10-01-preview/AutomationRules.json

```diff
@@ -495,9 +480,11 @@
         "IncidentDescription",
         "IncidentSeverity",
         "IncidentStatus",
-        "IncidentTactics",
         "IncidentRelatedAnalyticRuleIds",
+        "IncidentTactics",
+        "IncidentLabel",
         "IncidentProviderName",
+        "IncidentOwner",
```

Comment:
IncidentOwner shouldn't be here. it will be added in the API version after this one